sos/plugins/cifs.py

```diff
@@ -0,0 +1,30 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class Cifs(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """Cifs filesystem
+    """
+    packages = ('cifs-utils',)
+    plugin_name = "cifs"
+    profiles = ('storage',)
+
+    def setup(self):
+        self.add_copy_spec([
+            "/proc/fs/cifs"
+        ])
```

Comment:
A whole plugin for a single file is overkill: if there is nothing else to put in here then please add this to the existing `filesys` plugin instead.

Also, please when adding calls to `Plugin` class methods that accept either a single item (string), or a list of items (strings), do not use the list style to add a single element list. It should be just:

```
        self.add_copy_spec("/proc/fs/cifs")
```